libs/cart/src/drivers/interfaces/cart-order-service.interface.ts

```diff
@@ -0,0 +1,19 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { DaffCart } from '../../models/cart';
+import { DaffCartPaymentMethod } from '../../models/cart-payment';
+
+/**
+ * The interface responsible for managing a customer's cart.
+ */
+export interface DaffCartOrderServiceInterface<T extends DaffCart = DaffCart> {
+	/**
+	 * Place an order.
+	 */
+  placeOrder(id: T['id'], payment?: DaffCartPaymentMethod): Observable<T>;
```

Comment:
Not sure what the return should be. If we make it a `DaffOrder` object then that imposes a dep from cart onto checkout.

Reply:
I think we should return a `string | number` for the return type (it would be the `id` of the resultant order).

Reply:
@damienwebdev so return `{id: string | number}` like we did with `CartService#create`?

Reply:
Yes, I think that makes sense.

Reply:
done